source/app/store/entities/tracks/data.js

```diff
@@ -23,8 +23,28 @@ export const tracksSchema = new schema.Array(trackSchema)
 // Track Lenses
 export const lensTracks = lensPath(['entities', 'tracks'])
 export const lensTrack = id => lensPath(['entities', 'tracks', id])
+
+const esModeDescription = `In this track, you can search through multiple documents and find information faster
```

Comment:
You might need to update this since we have comprehend medical decoupled too.
https://github.com/awslabs/document-understanding-solution/blob/fb9d97ce86e95da9bf40546e0f4fd0ad12833a47/source/app/store/entities/tracks/data.js#L27

Reply:
Yep, this wasnt merged in when I opened the PR. It seems easier to merge and rebase off the changes.
Opened a separate PR and branch :https://github.com/awslabs/document-understanding-solution/pull/161/